interfaces/builtin/mpris.go

```diff
@@ -221,14 +224,40 @@ func (iface *MprisInterface) ConnectedSlotSnippet(plug *interfaces.Plug, slot *i
 	}
 }
 
+func (iface *MprisInterface) GetName(attribs map[string]interface{}) (string, error) {
+	// default to snap name if 'name' attribute not set
+	mprisName := "@{SNAP_NAME}"
+	for attr := range attribs {
+		if attr != "name" {
+			return "", fmt.Errorf("unknown attribute '%s'", attr)
+		}
+		name, ok := attribs[attr].(string)
```

Comment:
Well, I set the default up above in mprisName and only modify it if 'name' is an attribute, so it seems equivalent? Maybe I'm missing the context for your comment. Can you explain your reservation to the current implementation?